services/src/main/java/org/keycloak/protocol/AuthorizationEndpointBase.java

```diff
@@ -131,4 +133,17 @@ protected AuthenticationFlowModel getAuthenticationFlow() {
         return realm.getBrowserFlow();
     }
 
+    protected void checkSsl() {
```

Comment:
I'm using it in the DockerEndpoint.  I thought it would be useful there, as it appears in some form in AuthorizationEndpoint and SamlService (both child classes of this).  I didn't want to get too crazy with the refactoring since I'm not supposed to touch anything that's not strictly related to Docker Protocol.  Can revert back to a copy-paste if that's preferred.